misc/python/materialize/cli/ci_logged_errors_detect.py

```diff
@@ -260,6 +264,28 @@ def get_error_logs(log_files: list[str]) -> list[ErrorLog]:
                 ):
                     continue
                 error_logs.append(ErrorLog(match.group(0), log_file))
+            open_panics = {}
+            for line in iter(data.readline, b""):
+                line = line.rstrip(b"\n")
+                if match := PANIC_START_RE.match(line):
```

Comment:
Is it safe to assume that the expression always starts at the beginning of the line? Otherwise, consider using `search` instead of `match`.

Reply:
Yes, should always be at the start. If we start interleaving characters on individual lines, all of our logging will be broken.

Reply:
Alright 👍